frame/dapps-staking/src/benchmarking.rs

```diff
@@ -180,22 +195,37 @@ benchmarks! {
         assert_last_event::<T>(Event::<T>::Withdrawn(staker, unstake_amount).into());
     }
 
-    claim {
-        let n in 2 .. T::MaxNumberOfStakersPerContract::get();
+    claim_staker {
+        initialize::<T>();
+        let (_, contract_id) = register_contract::<T>()?;
+
+        let number_of_stakers = 5;
+        let claim_era = DappsStaking::<T>::current_era();
+        let stakers = prepare_bond_and_stake::<T>(number_of_stakers, &contract_id, SEED)?;
+        let staker = stakers[0].clone();
+        advance_to_era::<T>(claim_era + 1u32);
+
+    }: _(RawOrigin::Signed(staker.clone()), contract_id.clone())
+    verify {
+        let mut staker_info = DappsStaking::<T>::staker_info(&staker, &contract_id);
+        let (era, _) = staker_info.claim();
+        assert!(era > claim_era);
+    }
 
+    claim_dapp {
         initialize::<T>();
-        let (developer_id, contract_id) = register_contract::<T>()?;
+        let (developer, contract_id) = register_contract::<T>()?;
 
-        let number_of_stakers = n - 1;
+        let number_of_stakers = 5;
```

Comment:
Can you tell me why the number of stakers are set as a constant?

Reply:
Leftover from previous implementation when weight was dependent on the number of active stakers.
I could change it to e.g. 1 or just remove this (immutable) variable.